### DISCUSSION TITLE
ValueError: Old style callback was removed in version 1.6.  See: https://xgboost.readthedocs.io/en/latest/python/callbacks.html.

### DISCUSSION THREAD

**Discussion by lizexu123** [mlc-ai/mlc-zh#84] [General]
是不是我的ubuntu里面xgboost版本高于了1.6.0，应该降低下版本
![image](https://user-images.githubusercontent.com/39205361/181893222-657fbb49-19f9-4b03-b0ae-5d393fed9999.png)

**Comment by Hzfengsy** [comment]
建议降低版本到1.5.x，目前这个问题社区正在逐步修复

**Reply by lizexu123** [reply]
好的，谢谢
